src/presentation/ui/component/runtimes/phpRuntimeTabContent.templ

```diff
@@ -0,0 +1,268 @@
+package componentRuntimes
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+	"strconv"
+)
+
+script PhpRuntimeTabContentLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('php', () => ({
+			// Primary states
+			selectedVersion: '',
+
+			// Auxiliary states
+			selectedConfigType: 'modules',
+			setSelectedConfigType(configType) {
+				this.selectedConfigType = configType;
+			},
+			updateModuleStatus(vhostHostname, version, moduleName, moduleStatus) {
+				htmx.ajax(
+					'PUT',
+					'/api/v1/runtime/php/' + vhostHostname + '/',
+					{
+						swap: 'none',
+						values: {
+							version: version,
+							'modules.name': moduleName,
+							'modules.status': moduleStatus
+						},
+					},
+				);
+			},
+			updateSettingValue(vhostHostname, version, settingName, settingValue) {
+				htmx.ajax(
+					'PUT',
+					'/api/v1/runtime/php/' + vhostHostname + '/',
+					{
+						swap: 'none',
+						values: {
+							version: version,
+							'settings.name': settingName,
+							'settings.value': settingValue
+						},
+					},
+				);
+			},
+
+			// Modal states
+			isUpdatePhpVersionModalOpen: false,
+			openUpdatePhpVersionModal(version) {
+				this.selectedVersion = version;
+				this.isUpdatePhpVersionModalOpen = true;
+			},
+			closeUpdatePhpVersionModal() {
+				this.selectedVersion = '';
+				this.isUpdatePhpVersionModalOpen = false;
+			},
+			updateVersion(vhostHostname) {
+				htmx.ajax(
+					'PUT',
+					'/api/v1/runtime/php/' + vhostHostname + '/',
+					{
+						swap: 'none',
+						values: { version: this.selectedVersion },
+					},
+				);
+				this.closeUpdatePhpVersionModal();
+			},
+		}));
+	});
+}
+
+templ PhpRuntimeTabContent(
+	runtimeOverview presenterDto.RuntimeOverview,
+	existentVhostsHostnames []string,
+) {
+	<!-- PhpRuntimeTabContent -->
+	@PhpRuntimeTabContentLocalState()
+	<div class="bg-os-800 -mt-4 rounded-b-xl rounded-r-xl p-4" x-data="php">
+		<div class="bg-os-500 rounded-b-lg rounded-r-lg px-4 py-2">
+			if runtimeOverview.IsInstalled {
+				<div class="p-4">
+					<div class="float-left w-1/5">
+						@componentForm.SelectInput(
+							"virtualHost", "Virtual Host Hostname",
+							"", runtimeOverview.VirtualHostHostname.String(),
+							"changeSelectedVhostHostname($event.target.value)",
```

Comment:
When an option is selected in the `<select>`, an event is emitted, carrying the value of the selected option. To access it, you use `$event.target.value`. Ideally, you would pass a callable function and, within the component, call that function, passing `$event.target.value` directly to it as a param. However, this would make it impossible to use other functions that require more than one param, as we have examples here during the update of modules and settings.

To improve readability, I decided to assign `$event.target.value` to a constant and pass it into the function. However, I feel this approach is too vague and becomes a requirement every time the `<select>` is used. Do you have any ideas on how to work around this?